clojure-ts-mode.el

```diff
@@ -93,6 +93,12 @@ itself."
   :type 'boolean
   :package-version '(clojure-ts-mode . "0.2.0"))
 
+(defcustom clojure-ts-toplevel-inside-comment-form nil
+  "Eval top level forms inside comment forms instead of the comment form itself."
+  :type 'boolean
+  :safe #'booleanp
+  :package-version '(clojure-ts-mode . "0.3.0"))
```

Comment:
You're too fast. After I wrote my remark I remembered that this package used a different versioning from what I normally do. :D "0.2.1" would be better.

Reply:
ok, reverted :)